_data/lsif-0-4-0-toc.yml

```diff
@@ -50,8 +50,8 @@
       anchor: resultRanges
     - title: Meta Data
       anchor: metaData
-    - title: Emitting Contstraints
-      anchor: emittingContstraints
+    - title: Emitting Constraints
+      anchor: emittingConstraints
```

Comment:
I'd strongly encourage at least fixing the human readable content if not the anchors...